data/importers/page.md

```diff
@@ -200,24 +200,175 @@ For each importer, we will cover only columns of interest.
 * The ICAO or IATA code for the arrival airport.
   *Note: you should only use one of ID, ICAO or IATA when adding routes and the Import modal will ask which you wish to use. If you're using the ID, this field should be blank*
 ---
-* Start Date / End Date
-* Date and Time when the route starts/ends. End Date can be blank; Start Date can be any day in the past or the future. Format for both fields is yyyy-mm-dd hh:mm - please ensure you set correct formatting in your spreadsheet editor.
+* Callsign
+* Callsign for the route in the ICAO format. 
+  *Note: Must be a valid combination of ICAO and IATA code from the Airline Callsign Parameters section in the Orwell Settings -> VDS*
+---
+* Flight Number
+* Flight Number for the route in the IATA format. 
+  *Note: Must be a valid combination of ICAO and IATA code from the Airline Callsign Parameters section in the Orwell Settings -> VDS*
 ---
 * Fleets
 * Comma separated list of Fleet IDs - 5, 15, 25, 30. IDs can be retrieved from Export or VDS.
 ---
+* Altitude
+* Cruise altitude in the feet format (e.g. 35000).
```

Comment:
Just for the sake of completeness - it will also accept FL350, /350 and 350. All will convert to 35000. Similar with FL080 and 080 for 8000. Do you think it's worth adding that?